src/pages/_document.js

```diff
@@ -1,13 +1,18 @@
 import Document, { Html, Head, Main, NextScript } from 'next/document'
 
 class AppDocument extends Document {
+  static async getInitialProps(ctx) {
+    const initialProps = await Document.getInitialProps(ctx)
+    return { ...initialProps }
+  }
+
```

Comment:
How come this was added? I couldn't see how it's related to styling changes..

Reply:
I've compared it from Hackney Design System [examples][1]

[1]:https://github.com/LBHackney-IT/hackney-design-system-examples/blame/main/next/pages/_document.js